content/ending-frozen-applications.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Ending Frozen Applications and Desktops
+description: >
+  Linux provides several methods to kill frozen or hung up processes and applications using GUI applications or terminal commands.
+keywords:
+  - frozen applications
+  - frozen desktop
+  - frozen terminal
+  - hung up
+  - not responding
+  - System76
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section:
+---
+
+## Ending Frozen Processes, Applications, and Desktops
+
+Linux offers many methods to end hanging or frozen processes. The `gnome-system-monitor` provides a GUI process management application with features similar to the Windows Task Manager, while the `kill` command can end single or multiple processes from the Terminal.
+
+>Note: This article details several methods to end processes for a given scenario, but is not exhaustive or guaranteed to provide the most appropriate method for every scenario. Users should also keep in mind that suddenly terminating processes can result in unwanted data loss or system stability issues.
+
+## End Specific Applications or Windows
+
+### Using gnome-system-monitor (GUI)
+
+The gnome-system-monitor comes prepackaged in Pop!_OS. This application provides a GUI interface and functionality similar to the Windows Task Manager.
+
+1. Press <kbd>Super</kbd> + <kbd>T</kbd> to launch a Terminal session.
+2. Enter `gnome-system-monitor` and hit <kbd>Enter</kbd>.
+3. Locate frozen and hanging applications.
+
+   ![gnome-system-monitor](/images/ending-frozen-applications/gnome-system-monitor)
+
+4. Right click the application, click Stop in the dropdown menu.
+
+### Using the kill Command (CLI)
+
+The `kill` command can send a signal termination (SIGTERM) command to any running processes. This process requires knowing the process IDs (PID) of the application(s) to be terminated. The `pidof` command will display the PID(s) associated with any running process.
+
+1. Press <kbd>Super</kbd> + <kbd>T</kbd> to launch a Terminal session.
+2. Type pidof + process or application name and hit Enter.
+
+   ```bash
+   pidof firefox
+   ```
+
+3. Take note of the listed PID(s).
+4. Type kill + PID(s) to kill all associated PID(s), then hit <kbd>Enter</kbd>.
+
+    ```bash
+    kill 20869 20862
+    ```
+
+Tip: Use the pipe <kbd>|</kbd> and `xargs` command to pipe multiple PIDs as arguments into the kill command:
```

Comment:
This is a handy trick. Is there any reason a user would need to list and copy/paste the PIDs as described above instead of just doing this in the first place?

Reply:
yea good point